_posts/2021-03-09-stellar-issue-asset.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: How to issue an asset on Stellar
+subtitle: AFFEN goes live
+tags: [poc, blockchain, stellar, token, asset, github]
+author: gmzabos
+---
+
+## Intro - what is Stellar?
+
+[Stellar](http://stellar.org/){:target="_blank"} is an open-source network for currencies and payments. It is public, there is no "real" owner as it is a decentralized network. See the [Intro](https://stellar.org/learn/intro-to-stellar){:target="_blank"} on their website, as it speaks for itself better than i could do.
+
+## Intro - why do we issue our own asset?
```

Comment:
I would prefer to have a section like `what is an asset` or is this knowledge what can be assumed as present?

Reply:
I'll skip this one. The intro explains it.